gtsfm/bundle/bundle_adjustment.py

```diff
@@ -136,6 +145,37 @@ def create_computation_graph(self, sfm_data_graph: Delayed) -> Delayed:
         return dask.delayed(self.run)(sfm_data_graph)
 
 
+def aggregate_ba_stats(ba_data: GtsfmData) -> Dict[str,Any]:
+    """Create a dictionary of summary statistics for a bundle adjustment result.
+
+    Args:
+        ba_data: bundle adjustment result
+
+    Returns:
+        dictionary containing summary statistics of bundle adjustment result
+    """
+    track_lengths_3d = ba_data.get_track_lengths()
+    scene_reproj_errors = ba_data.get_scene_reprojection_errors()
+
+    convert_to_rounded_float = lambda x: float(np.round(x,3))
```

Comment:
why is this necessary? Arent they float already? Maybe the rounding could be done without a lambda?

Reply:
It's a bit weird -- np.round() or np.mean() will return a float32 object that is not json-serializable, until we cast it to a python `float`. So I couldn't get it to work without this